src/status_im/common/floating_button_page/view.cljs

```diff
@@ -53,7 +53,7 @@
 
 (defn view
   [{:keys [header footer customization-color footer-container-padding header-container-style
-           gradient-cover?]
+           gradient-cover? top?]
```

Comment:
`top?` is a bit short of information here - is it for the header, footer or the content? 
So maybe we use `content-top?` or perhaps we should instead just pass header-style, footer-style and content-container-style?

Reply:
it's for the entire component, and because of gradient-cover it's not a container style or whatever, it's a flag for top safe-area, we use this in the other parts of code base